app/templates/main/about.html

```diff
@@ -0,0 +1,19 @@
+{% extends 'layouts/base.html' %}
```

Comment:
In the future, when adding a completely new page (like the about page) let's do it on a separate branch